mobile-ui/src/main/java/com/playone/mobile/ui/BufferooApplication.kt

```diff
@@ -23,7 +24,9 @@ class BufferooApplication : DaggerApplication() {
 
     private fun setupTimber() {
         if (BuildConfig.DEBUG) {
-            Timber.plant(Timber.DebugTree())
+            plant{
+                debugTree()
```

Comment:
The coding style is difference from the others.
```kotlin 
plant { debugTree() }
```
It could be better!!

Reply:
fixed